wagtail_localize_rws_languagecloud/forms.py

```diff
@@ -64,16 +65,16 @@ def __init__(
             widget=forms.Select(),
         )
 
-    @property
+    @classproperty
```

Comment:
I changed this to `classproperty` because I needed to access the static values without creating an instance of this form.

Reply:
This also doesn't have to be a blocker, but getting these from properties on `LanguageCloudProjectSettingsForm` in a management command is a bit wierd. I reckon it would make more sense to move these out to standalone settings helper functions which can be called both in `update_translated_pages` and in `LanguageCloudProjectSettingsForm`. I think we can punt it to an issue as a bit of tech debt to pay off if we don't want to do it here.

Reply:
You're right. I'll refactor this to be less icky.

Reply:
I've refactored this so that the `default_*` methods are now standalone functions.